controllers/tests/controller_server/host_definer/settings.py

```diff
@@ -0,0 +1,69 @@
+from controllers.tests.common.test_settings import HOST_NAME
+
+NAME = 'name'
+NAMESPACE = 'namespace'
+SPEC = 'spec'
+METADATA = 'metadata'
+STATUS = 'status'
+DRIVERS = 'drivers'
+NODE_ID_FIELD_IN_CSI_NODE = 'nodeID'
+CSI_PROVISIONER_NAME = 'block.csi.ibm.com'
+FAKE_SECRET = 'fake_secret'
+FAKE_SECRET_NAMESPACE = 'fake_secret_namespace'
+FAKE_SECRET_ID = (FAKE_SECRET, FAKE_SECRET_NAMESPACE)
+FAKE_NODE_NAME = 'fake_node_name'
+FAKE_DAEMON_SET_NAME = 'fake_daemon_set_name'
+FAKE_NODE_PODS_NAME = '{}-jdf5g'.format(FAKE_DAEMON_SET_NAME)
+FAKE_SECRET_ARRAY = 'management_address'
+FAKE_SECRET_PASSWORD = 'fake_password'
+FAKE_SECRET_USER_NAME = 'fake_user_name'
+FAKE_STORAGE_CLASS = 'fake_storage_class'
+IQN = 'iqn.1994-05.com.redhat:686358c930fe'
+FAKE_NODE_ID = '{};;;{}'.format(IQN, HOST_NAME)
+FAKE_CSI_PROVISIONER = 'fake_csi_provisioner'
+TRUE_STRING = 'true'
+LABELS = 'labels'
+FAKE_LABEL = 'FAKE_LABEL'
+MANAGE_NODE_LABEL = 'hostdefiner.block.csi.ibm.com/manage-node'
+FORBID_DELETION_LABEL = 'hostdefiner.block.csi.ibm.com/do-not-delete-definition'
+WATCHER_HELPER_PATH = 'controllers.servers.host_definer.watcher.watcher_helper'
+NODES_WATCHER_PATH = 'controllers.servers.host_definer.watcher.node_watcher'
+SECRET_WATCHER_PATH = 'controllers.servers.host_definer.watcher.secret_watcher'
+CSI_NODE_WATCHER_PATH = 'controllers.servers.host_definer.watcher.csi_node_watcher'
+STORAGE_CLASS_WATCHER_PATH = 'controllers.servers.host_definer.watcher.storage_class_watcher'
+SETTINGS_PATH = 'controllers.servers.host_definer.settings'
+RESOURCE_VERSION = 'resource_version'
+FAKE_RESOURCE_VERSION = '495873498573'
+FAKE_UID = '50345093486093'
+TYPE = 'type'
```

Comment:
what is it?
something like `TYPE_KEY`?

Reply:
done

Reply:
is it a general field of many things? if not, can we specify?

Reply:
done